### ISSUE TITLE
Parallax

### ISSUE THREAD

**Issue by Denperidge** [MaxxxFoxxx/4Claws-Website#3]
Parallax scrollen ziet er best geweldig uit. Misschien moeten we die shit ook toevoegen aan onze website!
Wat denken jullie?
[Voorbeeld](https://www.w3schools.com/howto/tryhow_css_parallax_demo.htm)
[W3Schools tutorial](https://www.w3schools.com/howto/howto_css_parallax.asp)

**Comment by MaxxxFoxxx** [comment]
Oke, ik snap, de foto's veranderen, klein probleem, we hebben geen achtergrond foto's, wa wilt ge dat veranderd als ge scrolled?

**Comment by Denperidge** [comment]
Allez, die verandere niet. Het zijn er meerdere onder elkaar, ma die foto's staan zo precies stil tot ze veranderen. Best chill

**Comment by Denperidge** [comment]
Maar om op je vraag te antwoorden, screenshots van onze games zouden nice zijn bijvoorbeeld

**Comment by MaxxxFoxxx** [comment]
oke, maar
we hebben ten eerste geenscreenshots, ten tweede, er staat like niks op onze homepage
ik vind het een cool idee, maar ik denk gewoon niet dat het zo practise gaat zijn

**Comment by Denperidge** [comment]
Fair enough. We kunnen dit eventueel later maken als we eenmaal wat verder zitten

**Comment by MaxxxFoxxx** [comment]
We zullen het oppenlaten voor discussie, maar idd voor later
